src/Speck.js

```diff
@@ -134,13 +134,32 @@ class Speck {
     return JSON.parse(JSON.stringify(rawData));
   }
 
+  toJSONContext(context) {
```

Comment:
Just one small thing: this function shouldn't be named `toJSONByContext` instead of `toJSONContext`?